tests/regression/63-affeq/19-witness.c

```diff
@@ -0,0 +1,18 @@
+//SKIP PARAM: --set ana.activated[+] affeq --set sem.int.signed_overflow assume_none --set ana.relation.privatization top --enable witness.yaml.enabled
+// Identical to Example 63/01; additionally checking that writing out witnesses does not crash the analyzer
+#include <goblint.h>
+
+void main(void) {
+    int i;
+    int j;
+    int k;
+    i = 2;
+    j = k + 5;
+
+    while (i < 100) {
+        __goblint_check(3 * i - j + k == 1);
```

Comment:
This invariant doesn't appear in `witness.yml` though.

Reply:
That is another unrelated issue. The invariant is given as `i#318-1/3j#319+1/3k#320+4/3=0` which then likely does not survive conversion to a CIL expression. I suggest we open a new issue, as this is a general problem of `relationAnalysis`, which I won't have time to address any time in the next two weeks.

No longer crashing here is a step in right direction in the meanwhile.

Reply:
I created a new issue for this.